src/riscv-integration.adoc

```diff
@@ -161,6 +161,15 @@ Capability load and store instructions also cause load or store/AMO address
 misaligned exceptions if the address is not naturally aligned to a CLEN
 boundary.
 
+Misaligned capability loads and stores are fatal errors.  Implementations must
+generate exceptions for misaligned capability loads and stores even if they
+allow misaligned integer loads and stores to complete normally.  Execution
+environments must report misaligned capability loads and stores as errors
+and not attempt to emulate them using byte access.  The Zicclsm extension
```

Comment:
I wonder if we want an explanatory note here as well along the lines of "Capability loads must also fetch the associated tag bit and there is only one tag bit per capability-size memory region, so a misaligned access that straddles multiple tag bit regions cannot be emulated in software."

Reply:
Not fully faithfully, but CHERI does a lot of "do something close enough, and then clear the tag bit" and I thought for a while that applied here too.

Reply:
I agree with @arichardson: it would be helpful to add a note for the rationale behind this paragraph